core/src/test/scala/org/dbpedia/extraction/mappings/DraftMappingExtractorTest.scala

```diff
@@ -0,0 +1,337 @@
+package org.dbpedia.extraction.mappings
+
+import java.io.File
+
+import org.dbpedia.extraction.destinations.Quad
+import org.dbpedia.extraction.ontology.io.OntologyReader
+import org.dbpedia.extraction.ontology.{OntologyProperty, Ontology}
+import org.scalatest.{PrivateMethodTester, FlatSpec}
+import org.scalatest.Matchers
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import scala.collection.mutable.{Set => MutableSet, HashSet}
+import org.dbpedia.extraction.sources.{XMLSource, WikiPage, MemorySource}
+import org.dbpedia.extraction.wikiparser.{WikiParser, Namespace, WikiTitle}
+import org.dbpedia.extraction.util.Language
+
+/**
+  *
+  */
+@RunWith(classOf[JUnitRunner])
+class DraftMappingExtractorTest extends FlatSpec with Matchers with PrivateMethodTester {
+
+  "DraftMappingExtractor" should """return correct property id's P17, P18, P94, P41""" in {
```

Comment:
it should actually return ( (YM, ölkə, P17) , (YM, şəkil, P18) , (YM, gerb, P94) ,  (YM, bayraq, P41) , (YM, telefon kodu, P473) , (YM, nəqliyyat kodu, P395) ,  (YM, telefon kodu, P473) , (YM, sayt, P856)  )